articles/azure/azs-how-add-disks.md

```diff
@@ -0,0 +1,247 @@
+---
+title: How to add disks on Azure Stack
+description: How to add disks on Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Disk Management
+toc_sub3:
+toc_sub4:
+toc_title: How to add disks on Azure Stack
+toc_fullpath: Users/How To/Disk Management/azs-how-add-disks.md
+toc_mdlink: azs-how-add-disks.md
+---
+# Adding disks on Azure Stack
+
+## Overview
+
+In Azure Stack there are two types of disks, **Managed** and **Unmanaged** both these disks can be used on VM's as both OS and data disks.
+
+### Managed disks
+
+These disks simplify disk management for Azure IaaS VMs by managing the storage accounts associated with the VM disks. You only have to specify the size of disk you need, and Azure Stack creates and manages the disk for you.
+
+### Unmanaged disks
+
+These disks require that you create a storage account to store the disks. The disks you create are referred to as VM disks and are stored in containers in the storage account.
+
+## Using the portal to add new disks to VM's
```

Comment:
Remove apostrophe

Reply:
Maybe switch this around:
Adding disks to VMs using the Portal

Reply:
done